src/manager/commands/command_install/sc_component_manager_command_install.cpp

```diff
@@ -11,51 +11,93 @@
 
 #include "src/manager/commands/command_init/constants/command_init_constants.hpp"
 
+
 ScComponentManagerCommandInstall::ScComponentManagerCommandInstall(std::string specificationsPath)
-  : m_specificationsPath(std::move(specificationsPath))
+        : m_specificationsPath(std::move(specificationsPath))
 {
 }
 
-ExecutionResult ScComponentManagerCommandInstall::Execute(
-    ScMemoryContext * context,
-    CommandParameters const & commandParameters)
+/**
+ * @brief Check if components from specification is available
+ * @param context current sc-memory context
+ * @param componentsToInstall vector of components identifiers
+ * @return vector of available components
+ */
+ScAddrVector ScComponentManagerCommandInstall::GetAvailableComponents(ScMemoryContext *context,
+                                                                      std::vector <std::string> componentsToInstall)
 {
-  ExecutionResult result;
-  std::vector<std::string> componentsToInstall;
-
-  try
-  {
-    componentsToInstall = commandParameters.at(PARAMETER_NAME);
-  }
-  catch (std::exception const & exception)
-  {
-    // TODO: Implement install all components method
-    SC_LOG_INFO("No identifier provided, installing all to install components");
+    ScAddrVector availableComponents;
+    for (std::string const &componentToInstallIdentifier: componentsToInstall)
+    {
+        ScAddr componentAddr = context->HelperFindBySystemIdtf(componentToInstallIdentifier);
 
-    return result;
-  }
+        SC_LOG_DEBUG("Validating component \"" + componentToInstallIdentifier);
```

Comment:
Don't forget to close quotation.
```suggestion
        SC_LOG_DEBUG("Validating component \"" + componentToInstallIdentifier + "\"");
```